content/en/docs/contribute/style/style-guide.md

```diff
@@ -29,7 +29,10 @@ and representing feature state.
 
 ## Language
 
-Kubernetes documentation uses US English.
+Kubernetes documentation has been translated into multiple languages
+(see [here](https://github.com/kubernetes/website/blob/master/README.md#localization-readmemds)).
```

Comment:
There is a page on the website about localization.

```suggestion
(see [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)).
```

Reply:
👍

Reply:
I split the number of localized languages and the way of contribute. How do you think? Please see the update.